en/docs/guides/identity-verification/idvp-ui.md

```diff
@@ -0,0 +1,307 @@
+        },
+        {
+            "name": "timeout",
+            "displayOrder": 2,
+            "hint": "A timout value in seconds",
+            "placeholder": "Enter the timeout value in seconds",
+            "label": "Timeout",
+            "type": "number",
+            "required": true,
+            "dataComponentId": "idvp-edit-config-settings-timeout",
+            "defaultValue": "100",
+            "minValue": 2,
+            "maxValue": 120
+        },
+        {
+            "name": "enableSSL",
+            "displayOrder": 4,
+            "hint": "Check the box to enable SSL",
+            "label": "Enable SSL",
+            "type": "checkbox",
+            "dataComponentId": "idvp-config-settings-enable-ssl",
+            "defaultValue": false
+        },
+        {
+            "name": "enableEncryption",
+            "displayOrder": 5,
+            "hint": "Enables or disables the encryption",
+            "label": "Enable Encryption",
+            "type": "toggle",
+            "dataComponentId": "idvp-config-settings-enable-encryption",
+            "defaultValue": true
+        },
+        {
+            "name": "description",
+            "displayOrder": 7,
+            "hint": "The description of the Onfido IDVP",
+            "placeholder": "Enter the description",
+            "label": "Description",
+            "type": "text_area",
+            "required": false,
+            "dataComponentId": "idvp-config-settings-description",
+            "defaultValue":"A test description"
+        }
+    ]
+    }
+}
+```
+
+3. ### Create a template.json file
+
+The template.json file contains the template that is used to create the IDV provider. When making an API call to create the IDV provider, the template is used as the base of the request body. The relevant fields of the template are then overwritten by the inputs received from the user through the UI.
+
+**Structure**
+
+The template.json should have the following structure.
+```
+{
+    "Type": "< IDV Provider type >",
+    "Name": "< A default name for the IDV Provider >",
+    "description": "< A default description for the IDV Provider >",
+    "isEnabled": < Should the IDV Provider be enabled by default >,
+    "claims": [
+        {
+            "localClaim": "< Local claim that should be mapped to IDVP claim >",
+            "idvpClaim": "< Extenal claim >"
+        },
+        .... other default claim mappings ...
+    ],
+    "configProperties": [
+    {
+        "key": "< key of the config property >",
+        "value": "< default value for the config property >",
+        "isSecret": < should the property be stored as a secret  >
+    },
+    .... other default config properties ...
+    ]
+}
+```
+
+Important:
+
+Since the relevant fields of the template are overwritten by the values provided by the UI, the default values provided through the metadata.json will take precedence over the default values provided through the template for config properties.
```

Comment:
```suggestion
!!! tip "Important"
> Since the relevant fields of the template are overwritten by the values provided by the UI, the default values provided through the metadata.json will take precedence over the default values provided through the template for config properties.

```

Reply:
Resolved